src/metabase/public_settings.clj

```diff
@@ -257,8 +258,12 @@
   (deferred-tru "Allow using a saved question or Model as the source for other queries?")
   :type       :boolean
   :default    true
-  :visibility :authenticated
+  :setter     :none
+  :visibility :internal
   :export?    true
+  :getter     (fn enable-nested-queries-getter []
+                ;; only false if explicitly set `false` by the environment
+                (not= "false" (u/lower-case-en (env/env :mb-enable-nested-queries))))
```

Comment:
This works, so I won't block the PR, but it might be cleaner to wipe the setting with a migration and then just have this be a normal `:internal` setting without a custom getter? `:internal` settings are normally env var-only; the only difference here is that there might be a value in the app DB that we need to delete. Take it or leave it.

Reply:
yeah it would. sounds like that isn't a big deal though if we're going to start ignoring the DB value anyway? if a customer downgrades and needs this still set they can set it via env var